dag_in_context/src/lib.rs

```diff
@@ -211,6 +211,7 @@ pub fn egglog_test(
         prologue(),
         include_str!("schedule.egg"),
     );
+    eprintln!("{}", program);
```

Comment:
I think that printing the whole program is bad, both here and on line 225, because if multiple tests fail it pushes the actual errors off the screen, and also scrolling through the terminal to read egglog sucks. Can we maybe just add (or advertise if it already exists) a top-level command to dump the program, and remove these two prints?

Reply:
Good catch on this print, committed by mistake
Idk how to add a top-level command to print the program for rust tests. Would add if I knew how